src/trio/_core/_generated_io_kqueue.py

```diff
@@ -43,7 +43,7 @@ def current_kqueue() -> select.kqueue:
 
 def monitor_kevent(
     ident: int, filter: int
-) -> ContextManager[_core.UnboundedQueue[select.kevent]]:
+) -> ContextManager[MemoryReceiveChannel[select.kevent]]:
```

Comment:
Huh actually I'm just realizing this is potentially a compat break. I can work around this using a kwarg but I need to see if this currently warns or not...

Reply:
Private subclass which adds the unbounded queue methods but warns if you call them?